src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/CancellationTokenParametersMustComeLast.cs

```diff
@@ -120,5 +136,12 @@ public override void Initialize(AnalysisContext context)
                 SymbolKind.Method);
             });
         }
+
+        private static bool HasCallerInformationAttribute(IParameterSymbol parameter, ImmutableHashSet<INamedTypeSymbol> callerAttributes)
+        {
+            return parameter.GetAttributes().Any(
+                attribute => callerAttributes.Any(
+                    callerAttribute => SymbolEqualityComparer.Default.Equals(callerAttribute, attribute.AttributeClass)));;
+        }
```

Comment:
[Optional] You could rewrite as a expression-bodied method.

```suggestion
        private static bool HasCallerInformationAttribute(IParameterSymbol parameter, ImmutableHashSet<INamedTypeSymbol> callerAttributes)
            => parameter.GetAttributes().Any(
                attribute => callerAttributes.Any(
                    callerAttribute => SymbolEqualityComparer.Default.Equals(callerAttribute, attribute.AttributeClass)));;
```